plugins/plugin-github/che-plugin-github-factory-resolver/src/main/java/org/eclipse/che/plugin/github/factory/resolver/GithubURLParserImpl.java

```diff
@@ -90,7 +92,7 @@ public GithubUrl parse(String url) {
         .withRepository(repoName)
         .withBranch(branchName)
         .withSubfolder(matcher.group("subFolder"))
-        .withDockerfileFilename(".factory.dockerfile")
+        .withDevfileFilename(".devfile")
```

Comment:
I think it should be `Devfile` or `.Devfile`. 
Need to decide which name we use and maybe rename [Devfile](in https://github.com/redhat-developer/devfile/blob/master/petclinic.yaml) in [redhat-developer/devfile](https://github.com/redhat-developer/devfile) repository

Reply:
very good point @sleshchenko 
Maybe we can start with `Devfile` for iteration 1

Reply:
There is no single opinion. I was looking and testin on official @lord repo at 
https://github.com/che-incubator/chectl/  and there is `.devfile`

Reply:
:) @l0rd ? seems you're using .devfile so needs to clarify the final (or at least the starting name) :)

Reply:
`.devfile` is much better choice :-) 

Jokes apart can the match be case insensitive and support both `.devfile` and `devfile` (maybe in a separate PR)?

Reply:
i'm not sure case insensitive is easy. On github http request it may consist of several different requests